api/openapi.yaml

```diff
@@ -34,6 +34,139 @@ paths:
                 $ref: "./definition.yaml#/components/schemas/ExecuteDefinitionResponse"
         422:
           description: "trying to create a test with an already existing ID"
+  /transactions:
+    get:
```

Comment:
Based on the UI design, we are going to need one single endpoint that returns `tests` and `transactions` (should support search, pagination, etc.). We could keep the `get /transactions` endpoint and implement the unified one in a new PR.